util/rlp/fuzz/gen_untrusted_corpus.rs

```diff
@@ -0,0 +1,113 @@
+extern crate rlp;
+extern crate ethcore_bigint;
+
+use std::env;
+use std::error::Error;
+use std::io::prelude::*;
+use std::path::PathBuf;
+use std::fs::File;
+use rlp::RlpStream;
+use ethcore_bigint::prelude::{U128, U256, H64, H128, H160, H256, H512, H520, H2048};
+
+fn create_file(path: &PathBuf) -> File {
+    match File::create(path.as_path()) {
+        Err(why) => panic!("couldn't create {}: {}",
+                           path.display(),
+                           why.description()),
+        Ok(file) => file
+    } 
+}
+
+fn write_to_file(f: &File, rlp: &RlpStream) {
+    let mut g = f.clone();
+    match g.write_all(rlp.as_raw()) {
+        Err(why) => {
+            panic!("couldn't write to file: {}", why.description())
+        },
+        Ok(_) => println!("successfully wrote to file")
+    }
+
+}
+
+fn write_corpus(corpdir: &str, rlp: &RlpStream) {
+    // Read in base path to fuzzing corpus directory from `RLPCORPUS` environment var
+    let corp = env::var("RLPCORPUS").unwrap();
```

Comment:
> what is fuzzing corpus? does it need to be persistent?

the fuzzing corpus starts as an initial set of valid input data, and cargo-fuzz mutates the initial data into a larger, potentially crash-causing input. it needs to be somewhat persistent, but you could generate the corpus from scratch each time.

generating from scratch is computationally more expensive (depends on size of corpus), and some amount of state is lost in code penetration. cargo-fuzz also has a command `cmin` to minify the corpus.

> It will crash on everyone's machine, cause RLPCORPUS is no defined

True, I wasn't sure on the preferred way to do this. I'll change the path to a constant in the test, and use relative path so its all maintained in the `fuzz/corpus/<test-name>` directory.

> I see no reason for tests to permanently spoil my filesystem. can you use tempdir?

What about adding tempdir as an option, for testers that don't want to have a persistent fuzzing corpus? 

Tempdir definitely seems to be the right way to go for automated build tests, since they just need basic fuzzing coverage. 

Then in contexts that require deeper coverage, tests can still be run with a persistent corpus.